doc/source/cluster/kubernetes/user-guides/ray-k8s-setup.md

```diff
@@ -0,0 +1,81 @@
+(ray-kubernetes-setup)=
+
+# Set up a Ray + Kubernetes cluster
+
+This document contains recommendations for setting up a Ray + Kubernetes cluster for your organization.
```

Comment:
The Ray and Kubernetes ecosystem encompasses various aspects. Could you specify which setup instructions are covered by this document?

Reply:
It seems to be covered by:

```
This guide covers best practices for these deployment considerations:

* Where to ship or run your code on the Ray cluster
* Choosing a storage system for artifacts
* Package dependencies for your application
```

Reply:
should be addressed